app/views/shared/steps/_markdown_help.html.erb

```diff
@@ -1,14 +1,14 @@
 <%# sensible formatting not included because the PRE formats all whitespace %>
-<pre class="small">Paragraphs are separated by empty lines.
+<pre class="small markdown-example">Paragraphs are separated by empty lines.
 
-This is a list of bulleted links...
-
-* [Link text](/link/href/value)
-* [Link text](/link/href/value)
-
-This is a list of non-bulleted links...
+Non-bulleted links are used for sequential tasks.
 
 - [Link text](/link/href/value)
 - [Link text](/link/href/value)
 
+Bulleted links are used when users have to choose from a list of options:
+
+* [Link text](/link/href/value)
+* [Link text](/link/href/value)
```

Comment:
can you add an example that has context please?
and also one that has a bullet but no link `- Hello World` `* Hello World`

Reply:
I've added the new content-reviewed version.